src/obfl-specification.html

```diff
@@ -2759,6 +2760,25 @@ <h4 id="L21851">Example</h4>
   </tbody>
 </table>
   
+<h3 id="L2762">mark-capital-letters</h3>
+
+<p>The overall policy for marking capital letters should be determined by the
+implementing application, for example per job or in a system setting. A user's
+preference should be respected.</p>
+
+<p>The marking of capital letters specified by <code>mark-capital-letters</code>
+applies to all elements in its content unless overridden with another instance of
+<code>mark-capital-letters</code>. In particular, the empty value of
+<code>mark-capital-letters</code> is used on an element B to override a
+specification of <code>mark-capital-letters</code> on an enclosing element A,
+without specifying another marking policy. Within B, the user's
+preference for marking capital letters should be applied.</p>
```

Comment:
After reading this paragraph several times, I still don't get it. Can you give an example?